astrology.lic

```diff
@@ -35,6 +41,20 @@ class Astrology
     result.compact.select { |data| data['circle'] <= DRStats.circle }
   end
 
+  def check_astral
+    return unless DRStats.circle > 99
+    return unless @ap_training
+    return unless @ap_source
+    return unless @ap_destination
+
+    if UserVars.astral_plane_exp_timer.nil? || Time.now - UserVars.astral_plane_exp_timer > 3600
+      wait_for_script_to_complete('bescort', ['ways', @ap_destination])
```

Comment:
I'm not sure what happens, but bescort does a check to make sure you know the destination shard. So presumably this would call bescort twice, set the timer and move on